src/api/v1/contribute.rs

```diff
@@ -36,10 +38,12 @@ pub enum ContributeError {
     NotUsersTurn,
     #[error("contribution invalid: {0}")]
     InvalidContribution(#[from] CeremoniesError),
-    #[error("signature error: {0}")]
-    Signature(SignatureError),
+    #[error("our signature error: {0}")]
+    OurSignature(SignatureError),
```

Comment:
Please change to `ReceiptSignature` or something like that. `Our` is a bit unclear.

Reply:
Good call :) Renamed to `ReceiptSigning`.